tests/e2e/local/kind/install_prereqs_macos.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+SCRIPTPATH="$(cd "$(dirname "$0")" || exit ; pwd -P)"
+ROOTDIR="$(dirname "${SCRIPTPATH}")"
+# shellcheck source=tests/e2e/local/common_macos.sh
+source "${ROOTDIR}/common_macos.sh"
+
+check_homebrew
+
+echo "Update homebrew..."
+brew update
+
+install_curl
+
+install_docker
+
+echo """
+NOTE: When running kind on MacOS it is recommended that you have at least 4GB of RAM
+and disk space (these are estimates for a single node kind cluster) dedicated to the
+virtual machine (VM) running the Docker engine otherwise the Kubernetes cluster might
+fail to start up. More info check User Guide: https://kind.sigs.k8s.io/
+"""
+
+install_kubectl
+
+function check_and_install_golang() {
+	echo "Checking Golang is installed..."
+    if ! go help > /dev/null; then
```

Comment:
nit: indentation looks off

Reply:
Thanks for reviewing @gargnupur. There was a mix between spaces and tab. I got that changed to only spaces.